docs/basic-features/supported-browsers-features.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Browser support and which JavaScript features are supported by Next.js.
+---
+
+# Supported Browsers and Features
+
+Next.js supports **IE11 and all modern browsers** (Edge, Firefox, Chrome, Safari, Opera, et al) with no required configuration.
+
+## Polyfills
+
+We transparently inject polyfills required for IE11 compatibility. In addition, we also inject widely used polyfills, including:
+
+- [**fetch()**](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) — Replacing: `whatwg-fetch` and `unfetch`.
+- [**URL**](https://developer.mozilla.org/en-US/docs/Web/API/URL) — Replacing: the [`url` package (Node.js API)](https://nodejs.org/api/url.html).
+- [**Object.assign()**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) — Replacing: `object-assign`, `object.assign`, and `core-js/object/assign`.
+
+If any of your dependencies includes these polyfills, they’ll be eliminated automatically from the production build to avoid duplication.
+
+In addition, to reduce bundle size, Next.js will only load these polyfills for browsers that require them. The majority of the web traffic globally will not download these polyfills.
+
+### Server-side Polyfills
+
+In addition to `fetch()` on the client side, Next.js polyfills `fetch()` in the Node.js environment. You can use `fetch()` on your server code (such as `getStaticProps`) without using polyfills such as `isomorphic-unfetch` or `node-fetch`.
+
+### Custom Polyfills (Advanced)
+
+If your own code or any external npm dependencies require features not supported by your target browsers, you need to add polyfills yourself. In that case, you can take a look at the [with-polyfills](https://github.com/vercel/next.js/blob/canary/examples/with-polyfills/) example.
```

Comment:
I don't think we should advertise this example as it overrides internals. Ideally we'd recommend a more progressive way like "if you use a feature that is not supported in all browsers load in that component"

Reply:
Thanks, this was done here: https://github.com/vercel/next.js/pull/13766